persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -122,6 +126,17 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
     case GetState =>
       sender() ! state
 
+    case GetStatePaginated(page) =>
+      val tsErrState = state.tsParsingErrors
+      val tsErrorCount = if (tsErrState.isEmpty) 0 else 1
```

Comment:
Correct. According to the requirements of this story, each page should have 20 lines' errors (i.e. all errors from 20 lines of the hmda file), not 20 individual errors. So the TS's errors only add 1 to the count for that page.

I updated the variable name in dc0a6fce17c1f6d467f206d661e7f16af7a517b0.